saleor/webhook/payloads.py

```diff
@@ -281,6 +283,12 @@ def generate_product_payload(product: "Product"):
                 }
                 for media_obj in product.media.all()
             ],
+            "channel_listings": json.loads(
+                serialize_product_channel_listing_payload(
+                    product.channel_listings.all()
```

Comment:
I think we should prefetch `channel` here.

Reply:
We cannot add prefetch for deleted objects. I created ticket for optimalization.